src/Console/BaseCommand.php

```diff
@@ -225,7 +225,7 @@ protected function setOutputLevel(Arguments $args, ConsoleIo $io): void
      * @param \Cake\Console\ConsoleIo $io The console io
      * @return int|null|void The exit code or null for success
      */
-    abstract public function execute(Arguments $args, ConsoleIo $io);
+    abstract public function execute(Arguments $args, ConsoleIo $io): int|null|null;
```

Comment:
This was skipped because we can't declare a type with `|void` and most of these don't return anything.